src/containers/Party.tsx

```diff
@@ -49,9 +49,15 @@ class Party extends React.Component<any, any> {
 
 function mapStateToProps(state) {
     return {
-        party: state.party
+        party: state.party,
+        selfId: state.user.id,
+        hostId: state.party &&
+        state.party.users &&
+        state.party.users.find(u => u.host) &&
+        state.party.users.find(u => u.host).id
     };
 }
 
+const mapDispatchToProps = dispatch => bindActionCreators(ActionCreators as any, dispatch);
 
-export default connect(mapStateToProps)(Party);
\ No newline at end of file
+export default connect(mapStateToProps, mapDispatchToProps)(Party);
```

Comment:
> we should have a different place for creating connect objects and only export the class component here

What do you suggest? That every component have another file that defines what state it has mapped to its props?